runners/flink/runner/src/main/java/org/apache/beam/runners/flink/FlinkRunnerResult.java

```diff
@@ -86,4 +87,9 @@ public State waitUntilFinish() {
   public State waitUntilFinish(Duration duration) {
     throw new UnsupportedOperationException("FlinkRunnerResult does not support waitUntilFinish.");
   }
+
+  @Override
+  public MetricResults metrics() {
+    throw new UnsupportedOperationException("FlinkRunnerResult does not support metrics.");
```

Comment:
Use consistent wording across runners. "does not support metrics" vs. "not supported by [...] runner _yet_"